packages/common/src/store/wallet/slice.ts

```diff
@@ -72,14 +72,15 @@ const slice = createSlice({
       { payload: { amount } }: PayloadAction<{ amount: StringWei }>
     ) => {
       if (isNullOrUndefined(state.balance)) return
-      const existingBalance = new BN(state.balance)
-      state.balance = existingBalance
-        .add(new BN(amount))
-        .toString() as StringWei
+      const existingBalance = BigInt(state.balance) as AudioWei
```

Comment:
ideally `AUDIO()` instead of `as AudioWei`... though I wonder if the issue is I made the constructors take in AudioWei instead of just a BigInt?